lib/galaxy/authnz/managers.py

```diff
@@ -188,17 +215,19 @@ def try_get_authz_config(sa_session, user_id, authz_id):
             raise exceptions.ItemAccessibilityException(msg)
         return qres
 
-    def authenticate(self, provider, trans):
+    def authenticate(self, provider, implementation, trans):
```

Comment:
If I use `provider = 'psa'` and `implementation = 'google'`, as a result of calling `authnz/psa/google/login` instead of `authnz/google/psa/login`, I get a `redirect_uri: null`. Maybe a more self-explanatory message would be better instead.

Reply:
Yeah, will do